index.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="utf-8">
+        <meta name="viewport" content="width=device-width">
+        <title>Weather Report</title>
+        <link href="./styles/index.css" rel="stylesheet" type="text/css"/>
+    </head>
+    <body>
+        <div id="heading">
+            <h1 id="default-city"> Weather Report for Seattle</h1>
+        </div>
+        <div class="box-content">
+            <div class="section-container">
+                <div class="left-boxes">
+                    <h3>Current Temperature: </h3>
+                    <button id="tempUpButton">up</button>
+                    <p id="temp" class="yellow">65 F</p>
+                    <button id="tempDownButton">down</button>
+                </div>
+                <section class="left-boxes">
+                        <h2><br>Sky View!</h2>
+                        <p>Change the sky view!</p>
+                    <label>What a beautiful sky:</label>
+                    <select name="skies" id="sky-dropdown">
+                        <option value="sunny">Sunny</option>
+                        <option value="cloudy">Cloudy</option>
+                        <option value="rainy">Rainy</option>
+                        <option value="snowy">Snowy</option>
```

Comment:
There is a small bug here - the first option is "sunny", and the starting garden-sky doesn't match (line 44).  If the user attempts to choose "sunny" the first time, nothing happens (because "sunny" is already selected) and the sky remains cloudy.  There are a few ways to fix this!
Option 1: Inside of registerEventHandlers (or some other callback that runs when "DOMContentLoaded") call skyView("sunny")
Option 2: Add a blank option to the top of the options list
Option 3: Set the garden-sky default on line 44 to "☀️☀️☀️☀️☀️☀️"